src/main/webapp/WEB-INF/tags/instructor/course/courseEditCourseInfo.tag

```diff
@@ -42,8 +50,8 @@
             
             <div class="form-group">
                 <div class=" col-sm-12 align-center">
-                    <input type="submit" class="button" id="btnSaveCourse" name="btnSaveCourse"
-                            style="display:none;" value="Save Changes" onclick="return verifyCourseData();">
```

Comment:
Oh, right. It didn't seem to be part of this issue, so I just wanted to check. It's fine to remove it since that function doesn't exist